File: JavaProject/src/main/java/org/unimagdalena/herencia/Vehiculo.java
package org.unimagdalena.herencia;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Vehiculo {

    private String modelo;
    private String marca;
    private String color;
    private String placa;

    public void conducir(){
        System.out.println("El " + marca + " " + modelo + " está en movimiento.");
    }

}

File: Taller/encapsulamiento/Vehiculo.php
<?php

class Vehiculo {
  protected $marca;
  private $modelo;

  public function __construct($marca, $modelo) {
      $this->marca = $marca;
      $this->modelo = $modelo;
  }

  public function getMarca() {
      return $this->marca;
  }

  private function getModelo(){
    return $this->modelo;
  }
}